server/src/controllers/duplicate.controller.ts

```diff
@@ -15,4 +15,10 @@ export class DuplicateController {
   getAssetDuplicates(@Auth() auth: AuthDto): Promise<DuplicateResponseDto[]> {
     return this.service.getDuplicates(auth);
   }
+
+  @Post('/resolve')
```

Comment:
You should be :id/resolve and it should return a 200.

Although it is debatable if you should just use the existing delete assets and bulk update endpoints instead.